RadixWallet/Features/TransactionReviewFeature/TransactionReview+View.swift

```diff
@@ -167,6 +164,30 @@ extension TransactionReview {
 
 		private let shadowColor: Color = .app.gray2.opacity(0.4)
 
+		@ViewBuilder
+		private func header(_ proposingDappMetadata: DappMetadata.Ledger?) -> some SwiftUI.View {
+			VStack(alignment: .leading, spacing: .small3) {
+				HStack(spacing: .small2) {
```

Comment:
This still will always put a space between `Text(L10n.TransactionReview.title)` and `Spacer(minLength: 0)` which is not what you actually want. What you want is a space between `Text(L10n.TransactionReview.title)` and `DappThumbnail(.known(thumbnail), size: .medium)` but only when the latter is present. When it is not present, you want the title to have all the space. The way to do that is as I proposed before, to put padding inside the conditional, and make the HStack spacing 0.

Reply:
Sure I know how it works, I meant "useful" in the context of having very big font and small padding. I'll indulge you.